roles/volumes/storageclass/tasks/main.yaml

```diff
@@ -0,0 +1,13 @@
+- name: Create StorageClass
+  k8s:
+    kubeconfig: "{{ dojot_kubeconfig_file_path | default(omit) }}"
+    state: present
+    definition: "{{ item }}"
+  register: result
+  until: result.failed == 0
+  retries: 3
```

Comment:
I think you should change the value of `retries` to `30` tto follow the template of the other services.

Reply:
Resolvido